ios/REANodesManager.m

```diff
@@ -552,4 +585,31 @@ - (NSString *)obtainProp:(nonnull NSNumber *)viewTag propName:(nonnull NSString
   return result;
 }
 
+- (void)tryToFlushUpdateBuffer
+{
+  if (_componentUpdateBuffer.count == 0) {
```

Comment:
I think this var is modified from ui thread and is being read here from possibly any thread